policyengine_us/variables/gov/states/la/tax/income/credits/cdcc/la_cdcc.py

```diff
@@ -0,0 +1,13 @@
+from policyengine_us.model_api import *
+
+
+class la_cdcc(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Louisiana Child and Dependent Care Credit"
+    unit = USD
+    definition_period = YEAR
+    reference = "http://legis.la.gov/Legis/Law.aspx?d=101769"
+    defined_for = StateCode.LA
+
+    adds = ["la_cdcc_non_refundable", "la_cdcc_refundable"]
```

Comment:
Instead of this, go the other direction: make a formula that computes the value regardless of the refundability. Then use the la_cdcc_refundable_eligible bool plus `adds` to create the refundable variable, and a formula for the non-refundable one.